.github/scripts/generate_pytorch_test_matrix.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
```

Comment:
this file has a shebang but it is not executable; could you either remove the shebang or mark it as executable?